### ISSUE TITLE
Certain subtitle files will not load citing "'windows1252' is not a supported encoding name."

### ISSUE THREAD

**Issue by itsthejoker** [jellyfin/jellyfin#648]
**Describe the bug**
When starting certain movies (but not all!) subtitles will not play even if selected. Starting the docker container without the daemon flag has an error shown below and subtitles do not render as expected. Other movies with subtitles do render appropriately as expected.

**To Reproduce**
1. Grab this file: 
[Pan's.Labyrinth.2006.1080p.BluRay.x264-[YTS.AM].en.srt.zip](https://github.com/jellyfin/jellyfin/files/2777220/Pan.s.Labyrinth.2006.1080p.BluRay.x264-.YTS.AM.en.srt.zip) (Github won't allow me to just upload the .srt file, so it's in the zip)
2. Rename it so that it matches a given movie in your library (or, if you have the one it matches, put it there)
3. Rescan your library so that it appears as a valid subtitle track
4. Select subtitle track in GUI
5. Play movie
6. Observe error in logs, see that no subtitles are present in the movie.

**Expected behavior**
Subtitles for this specific title should play.

**Logs**
```
[00:37:25] [INF] HTTP GET http://192.168.1.243:8096/emby/Videos/7b158a5a-b27c-e1e2-3577-c074947f8d4b/7b158a5ab27ce1e23577c074947f8d4b/Subtitles/2/0/Stream.vtt. Host=192.168.1.243:8096, Connection=keep-alive, Origin=http://192.168.1.243:8096, User-Agent=Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36, DNT=1, Accept=*/*, Referer=http://192.168.1.243:8096/web/index.html, Accept-Encoding=gzip, deflate, Accept-Language=en-US,en;q=0.9
[00:37:25] [ERR] Error processing request
System.ArgumentException: 'windows1252' is not a supported encoding name. For information on defining a custom encoding, see the documentation for the Encoding.RegisterProvider method.
Parameter name: name
   at System.Text.EncodingTable.GetCodePageFromName(String name)
   at System.Text.Encoding.GetEncoding(String name)
   at Emby.Server.Implementations.TextEncoding.TextEncoding.GetEncodingFromCharset(String charset) in /repo/Emby.Server.Implementations/TextEncoding/TextEncoding.cs:line 179
   at MediaBrowser.MediaEncoding.Subtitles.SubtitleEncoder.GetSubtitleStream(String path, String language, MediaProtocol protocol, Boolean requiresCharset, CancellationToken cancellationToken) in /repo/MediaBrowser.MediaEncoding/Subtitles/SubtitleEncoder.cs:line 211
   at MediaBrowser.MediaEncoding.Subtitles.SubtitleEncoder.GetSubtitleStream(MediaSourceInfo mediaSource, MediaStream subtitleStream, CancellationToken cancellationToken) in /repo/MediaBrowser.MediaEncoding/Subtitles/SubtitleEncoder.cs:line 194
   at MediaBrowser.MediaEncoding.Subtitles.SubtitleEncoder.MediaBrowser.Controller.MediaEncoding.ISubtitleEncoder.GetSubtitles(BaseItem item, String mediaSourceId, Int32 subtitleStreamIndex, String outputFormat, Int64 startTimeTicks, Int64 endTimeTicks, Boolean preserveOriginalTimestamps, CancellationToken cancellationToken) in /repo/MediaBrowser.MediaEncoding/Subtitles/SubtitleEncoder.cs:line 149
   at MediaBrowser.Api.Subtitles.SubtitleService.Get(GetSubtitle request) in /repo/MediaBrowser.Api/Subtitles/SubtitleService.cs:line 229
   at Emby.Server.Implementations.Services.ServiceExecGeneral.GetTaskResult(Task task) in /repo/Emby.Server.Implementations/Services/ServiceExec.cs:line 114
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost appHost, IRequest httpReq, IResponse httpRes, ILogger logger, String operationName, CancellationToken cancellationToken) in /repo/Emby.Server.Implementations/Services/ServiceHandler.cs:line 96
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken) in /repo/Emby.Server.Implementations/HttpServer/HttpListenerHost.cs:line 654
[00:37:25] [INF] HTTP Response 400 to 192.168.1.165. Time: 47ms. http://192.168.1.243:8096/emby/Videos/7b158a5a-b27c-e1e2-3577-c074947f8d4b/7b158a5ab27ce1e23577c074947f8d4b/Subtitles/2/0/Stream.vtt
[00:37:25] [INF] HTTP POST http://192.168.1.243:8096/emby/Sessions/Playing. UserAgent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36
```

**System (please complete the following information):**
 - OS: Docker
 - Browser: Chrome
 - Jellyfin Version: 10.0.2.0

**Comment by cvium** [comment]
Duplicate of #254

**Comment by JustAMan** [comment]
Might be fixed by #598 which is going to happen in new `10.1.0` release (coming really soon).

**Comment by itsthejoker** [comment]
@cvium oh crap you're right, my apologies. I'll close this one. Cheers!
